admin/merge_pr.py

```diff
@@ -279,6 +285,26 @@ def transform(x):
     return None
 
 
+def jenkins_info_from_response(project):
+    """
+    Get the Jenkins job information from the Jenkins API response.
+
+    :param dict project: the API response as a dict.
+    :return (JenkinsResults, dict): The first element will
```

Comment:
:question: I'm not crazy about having tuples as return types (reading `maybe_retry_jobs`, specifically `lambda x: x[1][0] in [JenkinsResults.FAILED, JenkinsResults.NOTRUN]` was sort of tricky to figure out). But it seems that this fits the existing style and API, so probably not something to be fixed in this PR.